server/src/main/java/org/spine3/server/storage/EntityField.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2017, TeamDev Ltd. All rights reserved.
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package org.spine3.server.storage;
+
+import com.google.protobuf.Any;
+import com.google.protobuf.Message;
+import com.google.protobuf.Timestamp;
+import org.spine3.SPI;
+import org.spine3.base.Command;
+import org.spine3.base.Event;
+import org.spine3.protobuf.TypeUrl;
+import org.spine3.server.entity.Entity;
+
+/**
+ * A container for the storage fields used in multiple {@link Storage storages}.
+ *
+ * <p>Basic usage is:
+ * <ul>
+ *     <li>Storing custom {@link Entity} state fields.
+ *     <li>Storing custom {@link Event} and {@link Command} messages fields.
+ * </ul>
+ *
+ * @author Dmytro Dashenkov
+ * @see StorageField
+ */
+@SPI
+public enum EntityField implements StorageField {
+
+    /**
+     * A field representing a timestamp in seconds.
+     *
+     * @see Timestamp#getSeconds()
+     */
+    timestamp,
+
+    /**
+     * A field for storing the part of a timestamp representing the amount of nanoseconds which is less then
```

Comment:
If you're talking about the `timestamp` field, we store __seconds__, not `Timestamp` there. This is more convenient for queries.